khal/settings/utils.py

```diff
@@ -25,7 +25,10 @@
 import os
 from os.path import expanduser, expandvars, join
 
-import pytz
+try:
+    import zoneinfo as ZoneInfo
+except ImportError:  # I am not sure if this is correct for the backport
+    from backports import zoneinfo as ZoneInfo
```

Comment:
I am not sure if this is correct for the backport as I did not change it while I imported the module instead of a module inside zoneinfo for the try portion of the try except.